src/lib.rs

```diff
@@ -433,7 +515,10 @@ impl Contract {
         if target_community.handle == community.handle {
             self.communities.insert(&target_community.handle, &community);
         } else {
-            require!(self.communities.get(&community.handle).is_none(), "Community handle `{community.handle}` is already taken");
+            require!(
+                self.communities.get(&community.handle).is_none(),
+                "Community handle `{community.handle}` is already taken"
```

Comment:
I believe formatting strings do not work in `require!`, please double-check that it actually substitutes the `{community.handle}` with the value, and not just leaving it as "{community.handle}" text.

Reply:
better add a unit test to confirm this?

Reply:
@petersalomonsen Sure, that is even better

Reply:
You're right it does indeed not work